ovn_k8s/common/kubernetes.py

```diff
@@ -157,9 +157,9 @@ def get_service(server, namespace, service):
     if api_token:
         headers.update({'Authorization': 'Bearer %s' % api_token})
     if ca_certificate:
-        response = requests.get(url, verify=ca_certificate)
+        response = requests.get(url, headers=headers, verify=ca_certificate)
     else:
-        response = requests.get(url)
+        response = requests.get(url, headers=headers)
```

Comment:
This fixes a problem with PR#28 - which is not yet merged.
In this case the approach we usually follows is to squash the two commits and force update the branch for PR #28